pkg/forwarder/forwarder.go

```diff
@@ -166,16 +166,24 @@ type DefaultForwarder struct {
 
 // NewDefaultForwarder returns a new DefaultForwarder.
 func NewDefaultForwarder(keysPerDomains map[string][]string) *DefaultForwarder {
+	return NewDefaultForwarderWithOptions(keysPerDomains, true)
+}
+
+// NewDefaultForwarderWithOptions creates a new DefaultForwarder with the given configuration
+func NewDefaultForwarderWithOptions(keysPerDomains map[string][]string, enableHealthChecks bool) *DefaultForwarder {
```

Comment:
We should either have a `NewDefaultForwarderWithoutHealthCheck` that would call an internal function with `false` like `NewDefaultForwarder` does, or pass a options struct.

Right now we're not really passing options. Since we don't have that much option having too method `func NewDefaultForwarderWithOptions(keysPerDomains map[string][]string) *DefaultForwarder` and `func NewDefaultForwarder(keysPerDomains map[string][]string) *DefaultForwarder` should be fine.

WDYT ?